glsk/glsk-document-cim/src/test/java/com/powsybl/glsk/cim/CimGlskDocumentImporterTest.java

```diff
@@ -60,6 +61,15 @@ public void testGlskDocumentImportB45() {
         CimGlskDocument cimGlskDocument = CimGlskDocument.importGlsk(getResourceAsInputStream(GLSKB45TEST));
         List<GlskShiftKey> glskShiftKeys = cimGlskDocument.getGlskPoints().get(0).getGlskShiftKeys();
         assertFalse(glskShiftKeys.isEmpty());
+        assertEquals(4, glskShiftKeys.size());
```

Comment:
maybe remove the previous assertFalse then

Reply:
Yes!